x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/aggregate-metrics/100_field_caps.yml

```diff
@@ -0,0 +1,58 @@
+setup:
+  - skip:
+      version: " - 7.15.99"
```

Comment:
Should this be be 8.4?

Reply:
yes, it was left from copy-paste. changed it now.